module/data/item/templates/none-namegiver-power.mjs

```diff
@@ -43,6 +43,28 @@ export default class NoneNamegiverPowerData extends SystemDataModel {
                 integer: true,
                 label: "ED.Item.Power.damageStep"
             } ),
+            difficulty: new foundry.data.fields.SchemaField( {
```

Comment:
Mhh, I forgot: why isn't this a subclass of `Ability` if it has all the same fields?

Reply:
since powerStep and DamageStep are not in ability